modules/ob-about-buildrun.adoc

```diff
@@ -6,7 +6,7 @@
 [id="about-buildrun_{context}"]
 = BuildRun resource
 
-A `BuildRun` resource invokes a build on your cluster, similar to any cluster job or Tekton task run. The `BuildRun` resource represents a workload on your cluster, which results in a running pod. A `BuildRun` is the running instance of a build. It instantiates a build for execution with specific parameters on a cluster.
```

Comment:
@likhithj1996, here the TaskRun is referred in a generic way, that is, task run. So, no need to change its formatting. Please close the PR, the original formatting here is correct.

Reply:
Thanks for the explanation in simple and clear way. I agree and will be closing the PR and the JIRA ticket. Thank you.